Source/WebKit/WebProcess/WebPage/MomentumEventDispatcher.h

```diff
@@ -34,6 +34,7 @@
 #include <WebCore/FloatSize.h>
 #include <WebCore/PageIdentifier.h>
 #include <WebCore/RectEdges.h>
+#include <WebCore/ScrollTypes.h>
```

Comment:
Does `RectEdges<T>` need the type definition? Maybe you can get away with declaring `ScrollBounceBehavior` here?